.travis.yml

```diff
@@ -1,46 +1,26 @@
-# Config file .travis.yml
-
 language: python
 
 python:
-#    - "pypy" # not supported by odoo 8
-#    - "3.4"  # not supported by odoo 8
-#    - "3.3"  # not supported by odoo 8
-    - "2.7"
-#    - "2.6"  # not supported by odoo 8
+  - "2.7"
 
 env:
-    - ODOO="https://github.com/savoirfairelinux/odoo/archive/setuptools-addons.tar.gz"  # Temp until https://github.com/odoo/odoo/issues/185 or https://github.com/odoo/odoo/issues/441 is fixed
-#    - ODOO="https://github.com/odoo/odoo/archive/master.tar.gz"
-#    - ODOO="https://github.com/OCA/OCB/archive/master.zip"
+  - LINT_CHECK="1"
+  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
+  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  - VERSION="8.0" ODOO_REPO="odoo/odoo" UNIT_TEST="1" LINT_CHECK="0"
```

Comment:
To be honest, I'm not sure why we need that when we have the two lines before. That's how it's proposed in https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml, but I'm happy to remove this if you agree